spec/dummy/Gemfile.lock

```diff
@@ -322,4 +322,4 @@ DEPENDENCIES
   web-console
 
 BUNDLED WITH
-   1.11.2
+   1.11.2
```

Comment:
@jbhatab Something is not right. Removing bootstrap-sass (OK ,not used) but the change to gemfile.lock is not right.